### PR TITLE
Temporarily bring bitangent back in vertex shader to fix the screenshot test

### PR DESCRIPTION
**Pull request by jiaweig-amzn**
Signed-off-by: jiaweig <51759646+jiaweig-amzn@users.noreply.github.com>

## What does this PR do?

The calculated bitangent seems to be inconsistent from the vertex stream. Bring the bitangent back before further investigation to determine the correctness of the bitangent.

## How was this PR tested?

The test should pass with little difference.
eye

And for reference:
Before and after this change:
https://github.com/o3de/o3de/pull/12173
Calculated TBN in vertex shader:
BeforeTBNChange
Calculated TBN in pixel shader:
AfterTBNChange

### CODE REVIEW

**Review by antonmic — APPROVED** [review]

File: Gems/Atom/Feature/Common/Assets/Shaders/Materials/Eye/Eye_PixelGeometryEval.azsli [hunk 1]
```diff
@@ -81,6 +81,7 @@ PixelGeometryData EvaluatePixelGeometry_Eye(VsOutput IN, bool isFrontFace)
 
     float3 vertexNormal, vertexTangent, vertexBitangent;
     ConstructTBN(IN.normal, IN.tangent, objectToWorld, objectToWorldIT, vertexNormal, vertexTangent, vertexBitangent);
+    vertexBitangent = normalize(mul(objectToWorld, float4(IN.bitangent, 0.0)).xyz);
```
Comment (santorac) [inline Gems/Atom/Feature/Common/Assets/Shaders/Materials/Eye/Eye_PixelGeometryEval.azsli:84]: I think we should have a comment here to explain why we're doing this.

Comment (santorac) [reply Gems/Atom/Feature/Common/Assets/Shaders/Materials/Eye/Eye_PixelGeometryEval.azsli:84]: Also we should have an Issue to investigate further and remove the bitangent again if possible, or provide an explanation about why the bitangent is necessary.

Comment (jiaweig-amzn) [reply Gems/Atom/Feature/Common/Assets/Shaders/Materials/Eye/Eye_PixelGeometryEval.azsli:84]: Right. That's what I plan to do. I'll add a comment to explain.

Comment (jiaweig-amzn) [reply Gems/Atom/Feature/Common/Assets/Shaders/Materials/Eye/Eye_PixelGeometryEval.azsli:85]: https://github.com/o3de/o3de/issues/13159

**Review by santorac — APPROVED** [review]

**Review by santipaprika — APPROVED** [review]

**Pull request merged**